src/utils/array.js

```diff
@@ -0,0 +1,10 @@
+export const range = (a, b) => [...Array(b + 1).keys()].slice(a);
```

Comment:
Super stoked to have a `range` function here (why is this not part of JS in the first place?).

Would you be willing to add the ability to supply _just_ one argument too? I could see benefit in doing stuff like `range(5).map(n => console.log(n))`?

Reply:
Let's add that functionality when we have that use case. That complicates this function just a little bit, and we don't have a great reason to need a single argument range function quite yet. I do think its a perfectly valid idea.